src/planner/PlanNode.h

```diff
@@ -198,11 +209,11 @@ class SingleInputNode : public SingleDependencyNode {
 class BiInputNode : public PlanNode {
 public:
     void setLeft(const PlanNode* left) {
-        left_ = left;
+        setDep(0, left);
```

Comment:
0, 1 use const var to replace

Reply:
I think it's not necessary to define two const variables to represent left/right positions, it's simple enough to understand.